tests/token/erc20/test_ERC20Burnable.py

```diff
@@ -124,9 +131,74 @@ async def test_burn_from_zero_address(erc20_factory):
 
 @pytest.mark.asyncio
 async def test_burn_invalid_uint256(erc20_factory):
-    erc20, _ = erc20_factory
+    erc20, _, _ = erc20_factory
 
     await assert_revert(
         erc20.burn(INVALID_UINT256).invoke(),
         reverted_with="ERC20: amount is not a valid Uint256"
     )
+
+
+@pytest.mark.asyncio
+async def test_burn_from(erc20_factory):
+    erc20, account1, account2 = erc20_factory
+
+    await signer.send_transaction(
+        account1, erc20.contract_address, 'increaseAllowance', [
+            account2.contract_address,
+            *AMOUNT
+        ])
+
+    tx_exec_info = await signer.send_transaction(
+        account2, erc20.contract_address, 'burnFrom', [
+            account1.contract_address,
+            *AMOUNT
+        ])
+
+    assert_event_emitted(
+        tx_exec_info,
+        from_address=erc20.contract_address,
+        name='Transfer',
+        data=[
+            account1.contract_address,
+            ZERO_ADDRESS,
+            *AMOUNT
+        ]
+    )
```

Comment:
what do you think about splitting this into two different tests? that'd be consistent with `burn` whose storage effects and emitted events are tested separately

Reply:
yea easy enough